### ISSUE TITLE
massivejs on NodeJS v4.x

### ISSUE THREAD

**Issue by jfbaquerocelis** [robconery/massive-js#125]
Hello, may I ask when will support **massivejs** _node@4.1.1_, or if someone has already been working on **4.1.1** massive please tell me as it did.
Windows7 use and have already tried to **upgrade-ready** library but have not succeeded.

Thank you for reading :)

**Comment by dmfay** [comment]
Last week I upgraded our app to 4.1.1 and didn't notice any problems in my testing (the change _just_ got merged down to the development branch though, so caveat emptor). We are tracking Massive's master branch directly for now rather than specifying a version, but right now master and 2.0.9 are identical anyway.

**Comment by jfbaquerocelis** [comment]
OMG! :(
thats weird. I keep trying, but I always throws a **.log** with error.

**Comment by dmfay** [comment]
Can you post the output?

**Comment by jfbaquerocelis** [comment]
![err_massivejs](https://cloud.githubusercontent.com/assets/6578139/10178177/dcc2fc8e-66c3-11e5-8c57-9ec9b9ede43d.JPG)

I'm trying but do not let me upload the file :(
I will continue trying anyway, it is a log with __6000+__ lines of code, **.txt** step but does not want to load it.

That control an image to see if they have an idea, I think the problem is _deasync_.

**Comment by trentrichardson** [comment]
@jfbaquerocelis I had a similar error after updating to node 4.1.1.  I simply deleted my node_modules directory and re-ran `npm install`

**Comment by jfbaquerocelis** [comment]
@trentrichardson  any results, so I tried not **massivejs** installed.
I tried to install separately deasync ```npm install -g deasync``` globally, but not successful. I think the problem is **deasync**. For now I will work with **node@0.12.7**, but wait that the problem is resolved. Thank you :)

**Comment by trentrichardson** [comment]
@jfbaquerocelis Ok.  It could be the case I'm just not using all the features of massivejs, or not using the features you are in your project.

For anyone else wondering, I had an existing project I have been developing on node 0.12.  I upgraded to node 4.1.1 and got an error about deasync (I don't recall the exact error).  I simply deleted the node_modules directory and ran `npm install` to get a fresh set of dependencies.  After that massivejs has been working flawlessly.
